src/digest.rs

```diff
@@ -47,14 +45,13 @@ pub fn hex_to_bytes(text: &str, dest: &mut [u8]) -> Result<()> {
 
 const N_BYTES: usize = 32;
 pub const DIGEST_NAME: &'static str = "SHA256SUM";
-pub use crypto::sha2::Sha256 as DigestComputer;
 
 pub fn create() -> DigestComputer {
-    sha2::Sha256::new()
+    Default::default()
 }
 
 
-#[derive(Clone,Copy,Debug,Eq,PartialEq)]
```

Comment:
Hm, I don't remember removing it. Sorry I missed it. Initially I worked on the older version of the code. Maybe it wasn't there? Either way, I'll add it back